composer.json

```
@@ -13,7 +13,7 @@
         "google/protobuf": "^3.12.2",
         "guzzlehttp/promises": "^1.3",
         "guzzlehttp/psr7": "^1.7.0|^2",
-        "google/common-protos": "^1.0"
+        "google/common-protos": "^1.0|^2.0"
```

Nit: The [double pipe](https://getcomposer.org/doc/articles/versions.md#version-range) is recommended
```suggestion
        "google/common-protos": "^1.0||^2.0"
```